playbooks/tofino/setup_nodes_ae.yml

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2019 Cable Television Laboratories, Inc.
```

Comment:
Can you rename this playbook to something like ae-node-buld.yml as the playbook naming will imply building of the environment vs. setting up the AE node for testing. Also, can you move this down one directory called 'ae' like with setup_ae_centos.yml as well?